docs/internal_api/index.rst

```diff
@@ -88,10 +88,17 @@ Methods
 .. autosummary::
    :toctree: _autosummary
 
-   Grid.__init_grid_var_names__
-   Grid.__from_ds__
-   Grid.__from_vert__
-   Grid.__init_grid_var_attrs__
+   __init_grid_var_names__
+   __from_ds__
+   __from_vert__
+   __init_grid_var_attrs__
+   _build_edge_node_connectivity
+   _build_face_edges_connectivity
+   _build_node_faces_connectivity
+   _build_nNodes_per_face
+   _populate_cartesian_xyz_coord
+   _populate_lonlat_coord
```

Comment:
These appear to be duplicates with the following sections, can remove. 

https://github.com/UXARRAY/uxarray/blob/9314962d46993a41289b7fea019814ff25a47d41/docs/internal_api/index.rst?plain=1#L128-L132

https://github.com/UXARRAY/uxarray/blob/9314962d46993a41289b7fea019814ff25a47d41/docs/internal_api/index.rst?plain=1#L149-L150